src/features/todos.js

```diff
@@ -1,45 +1,31 @@
 import { createSlice } from "@reduxjs/toolkit";
-const initialStateValue = [
-  {
```

Comment:
It's quite hard to review your PR since. it contains content changes based on the old one. if you could raise a PR for a new branch which would contain all changes at once.